src/model/invoice.model.js

```diff
@@ -0,0 +1,40 @@
+import { Schema, model } from "mongoose";
+
+const invoiceSchema = new Schema({
+  id: {
+    type: Number,
+    required: true,
+    unique: true,
+  },
+  price: {
+    type: Number,
+    required: true,
+  },
+  created_at: {
+    type: Date,
+    default: Date.now,
+    required: true,
+  },
+  tp_status: {
+    type: String,
+    enum: ["ACTIVE", "INACTIVE"],
+    required: true,
+  },
```

Comment:
esto debe tener por defecto "ACTIVE"
```suggestion
  tp_status: {
    type: String,
    enum: ["ACTIVE", "INACTIVE"],
    required: true,
    default: "ACTIVE"
  },
```